Sources/BraintreeShopperInsights/BTEligiblePaymentsRequest.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+import PassKit
+
+/// The POST body for `v2/payments/find-eligible-methods`
+struct BTEligiblePaymentsRequest: Encodable {
+    
+    private let customer: Customer
+    private let purchaseUnits: [PurchaseUnit]
+    private let preferences = Preferences()
+    
+    enum CodingKeys: String, CodingKey {
+        case customer = "customer"
+        case purchaseUnits = "purchase_units"
+        case preferences = "preferences"
+    }
+    
+    struct Customer: Encodable {
+        let countryCode: String = "US"
+        let email: String?
+        let phone: Phone?
+        
+        enum CodingKeys: String, CodingKey {
+            case countryCode = "country_code"
+            case email = "email"
+            case phone = "phone"
+        }
+    }
+
+    struct PurchaseUnit: Encodable {
+        let payee: Payee
+        let amount = Amount()
+        
+        struct Amount: Encodable {
+            let currencyCode = "USD"
+            
+            enum CodingKeys: String, CodingKey {
+                case currencyCode = "currency_code"
+            }
+        }
+
+        struct Payee: Encodable {
+            let merchantID: String
+            
+            enum CodingKeys: String, CodingKey {
+                case merchantID = "merchant_id"
+            }
+        }
+    }
+    
+    struct Preferences: Encodable {
+        let includeAccountDetails = true
+        let includeVaultTokens = true
```

Comment:
I plan to remove `include_vault_tokens` on my PR as you mentioned it is not needed. And from testing I still got back Venmo and PayPal recommendations.

Reply:
Removed in 35cf4f54323c87ea86ba9cab1f0ea5b58cc1bbe5!